logger/Tests/Benchmarks/Benchmarks.cpp

```diff
@@ -0,0 +1,172 @@
+#include <benchmark/benchmark.h>
+#include "../../Include/Logger.h"
```

Comment:
Please try to fix it so you can just:

```suggestion
#include "Include/Logger.h"
```